src/lib/loaders/loaders_with_authentication/tracked_entity.ts

```diff
@@ -9,19 +9,23 @@ import { map, find, extend } from "lodash"
  * @param {string} paramKey the key that’s to be used for the list of IDs param that’s sent with the API request
  * @param {string} trackingKey the key that’s to be used for this entity, e.g. `is_followed` or `is_saved`.
  * @param {string} [entityKeyPath] an optional path to a nested entity
+ * @param {string} [entityIDKeyPath] an optional path to the IDs used to compare entities (defaults to `id`, but `_id` is sometimes useful).
  */
 const trackedEntityLoaderFactory = (
   dataLoader: (id: any) => Promise<any>,
   paramKey: string,
   trackingKey: string,
-  entityKeyPath?: string
+  entityKeyPath?: string,
+  entityIDKeyPath: string = "id"
```

Comment:
This is what I had to do to get `_id` working – it seemed like the best approach but I'm open to suggestions.

Reply:
Oo, interesting bug, and yes adding an 'id path' with that default seems absolutely perfect to me!